traitsui/qt4/check_list_editor.py

```diff
@@ -146,10 +168,34 @@ def create_control(self, parent):
 
         self._mapper = QtCore.QSignalMapper()
         self._mapper.mapped[str].connect(self.update_object)
+        self._connections_to_dispose.append(
```

Comment:
I see only one connection in this list and looks like it could be disconnected without the list, so maybe `_connections_to_dispose` is unnecessary?

Reply:
Yep, agreed. I reverted this back to just calling disconnect.
Thanks!